### ISSUE TITLE
Torus should not list Metamask as the provider

### ISSUE THREAD

**Issue by dkent600** [daostack/alchemy#1542]
When logged in with Torus on rinkeby, hovering over the account in the top right lists Metamask as the provider.  It should list "Torus" or something.

I created a ticket on this in Web3Modal:  https://github.com/Web3Modal/web3modal/issues/130#issuecomment-602942907

And Torus has submitted a PR on it:
https://github.com/Web3Modal/web3modal/pull/133

**Comment by dkent600** [comment]
The Web3Modal PR was merged, presumedly giving us the fix for this, but it breaks our code as `getProviderInfo` is not longer being exported.  Have contacted them about this via PM.

**Comment by orenyodfat** [comment]
any alternative they provide that ?

**Comment by dkent600** [comment]
> any alternative torus provide that ?

@orenyodfat I'm sorry I don't understand the question.  Can you rephrase?

FYI the method doesn't come from Torus, it comes from Web3Modal, and that is where it must come from.
